README.md

```diff
@@ -87,6 +89,67 @@ $ bin/tiproxy --config=conf/proxy.yaml
 $ mysql -h127.0.0.1 -uroot -P6000
 ```
 
+### Run in k8s
+
+1. Generate a self-signed certificate like above.
+
+2. Create secrets for session token signing certs.
+
+```shell
+$ kubectl create secret generic basic-sess -n $NAMESPACE --from-file=crt=cert.pem --from-file=key=key.pem
+```
+
+3. Create the TiDB cluster. An example of spec is as follows:
+
+```yaml
+---
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+    name: tc
+spec:
+    version: nightly
+    pd:
+        replicas: 1
+        baseImage: pingcap/pd
+    tidb:
+        replicas: 2
+        baseImage: pingcap/tidb
+        tlsClient:
+            enabled: true
+        config: |
+            graceful-wait-before-shutdown=10
+            [security]
+              session-token-signing-cert = "/var/sess/cert.pem"
+              session-token-signing-key = "/var/sess/key.pem"
```

Comment:
This is auto-set if `tlsCluster: true`, maybe notice that?

Reply:
What about additionalVolumes and additionalVolumeMounts? Will they be auto set?

Reply:
Yes, you don't need to do anything if `tlsCluster: true`.

But you will need to create 4 or 5 certs for each component...